ENG-36990: new leafs to /ncs:cluster/global-settings/passthrough/permit list (#35)
NSO-developer/nso-gitbook: administration/advanced-topics/layered-service-architecture.md

admin@rfs1% commit
```
{% endcode %}

> To allow the passthrough user to change into any user set the as-any-user leaf or to any group set the as-any-group leaf. Please use with care as setting these leafs will allow the lsa_passthrough_user to elevate privileges by changing to user admin group admin.

Reviewer: ```suggestion
To allow the passthrough user to change into any user or group, set `as-any-user` or `as-any-group` respectively. However, use these settings with care, as they allow the lsa_passthrough_user to elevate privileges, for example by changing to user admin/group admin.
```

> To allow the passthrough user to change into any user set the as-any-user leaf or to any group set the as-any-group leaf. Please use with care as setting these leafs will allow the lsa_passthrough_user to elevate privileges by changing to user admin group admin.

Reviewer 2: ```suggestion
To allow the passthrough user to change into any user, set the `as-any-user` leaf, or for any group, set the `as-any-group` leaf. Use this with care as setting these leafs will allow the `lsa_passthrough_user` to elevate privileges by changing to `user admin` / `group admin`.
```
